modules/acquia_cms_headless/modules/acquia_cms_headless_ui/acquia_cms_headless_ui.module

```diff
@@ -216,7 +215,9 @@ function acquia_cms_headless_ui_local_tasks_alter(array &$local_tasks) {
     // prefix accounts for the fact that Media module exposes the canonical
     // route conditionally based on a config switch, and therefore creates the
     // Edit and View tasks with a deriver (which is why it's prefixed).
-    elseif (preg_match('/^(media\.tasks:)?entity\.(\S*)\.canonical$/', $id, $matches)) {
+    // Pattern includes all routes with entity and ends with canonical, the API
+    // menu will be visible only on node entities to render the jsonapi data.
+    elseif (preg_match('/^(media\.tasks:)?entity\.(\S*(?<!user))\.canonical$/', $id, $matches)) {
```

Comment:
I don't think this statement is correct.
`the API menu will be visible only on node entities to render the jsonapi data.`

I can see it for taxonomy term, I think we should find root cause why its causing issue with user.
![image](https://github.com/acquia/acquia_cms/assets/63041696/13e8617b-587e-4dbf-b88f-0c462787d51d)

Reply:
Even i'm not 100% sure on this solution as this looks similar to what we did to exclude users route.

Reply:
@apathak18 - As per discussion, please try pattern to bypass if its `views_view:view.`